config/src/main/java/org/axonframework/config/AggregateConfiguration.java

```diff
@@ -57,4 +59,11 @@
      * @return the {@link SnapshotFilter} defined in this configuration
      */
     SnapshotFilter snapshotFilter();
+
+    /**
+     * Returns the CommandHandlerAggregateFactory {@link Function} defined in this configuration.
+     *
+     * @return the CommandHandlerAggregateFactory {@link Function} defined in this configuration.
+     */
+    Function<Object, A> commandHandlerAggregateFactory();
```

Comment:
Can you provide a bit more context on what the default implementation should do? Considering that is an interface without a state and the parent interface `ModuleConfiguration` does not provide a way to get the current Configuration context and/or state of the configuration.

It is technically a breaking change, yes, but only if someone actively wrote an entire `AggregateConfiguration` implementation replacement.

As this is just a getter, we can choose not to provide it at the interface level and only provide it in `AggregateConfigurer`

Reply:
I might make a wrong assumption, but I figured we could use the current aggregate creation approach inside the `AggregateAnnotationCommandHandler`, since that is what's already done right now. Hence, it would act as the default.

Reply:
And I agree with the fact the chances are slim somebody has their own `AggregateConfigurer` implementation. But, I'd like to stick to the standard of not introducing breaking changes as often as possible. I believe this is an option where we do not have to break others :-)

Reply:
Removed the method from the interface as it was only a nice to have anyway.

We can't provide a default implementation here to use the default constructor because at the interface level we don't have a `Class` object on which to invoke the constructor in a reflective way.

Reply:
Gotcha!